dbt_platform_helper/providers/io.py

```diff
@@ -1,18 +1,21 @@
-import os
-
 import click
 
 from dbt_platform_helper.platform_exception import PlatformException
-
-DEBUG = os.environ.get("DEBUG", False)
+from dbt_platform_helper.providers.environment_variable import (
+    EnvironmentVariableProvider,
+)
 
 
 class ClickIOProvider:
+    def __init__(self, env_provider=None):
```

Comment:
EnvironmentVariableProvider doesn't have any setup, so can just be defaulted here.
```suggestion
    def __init__(self, env_provider=EnvironmentVariableProvider()):
```